ts_src/lib/converter/input/tapKeySig.ts

```diff
@@ -0,0 +1,31 @@
+import { KeyValue, TapKeySig } from '../../interfaces';
+import { InputTypes } from '../../typeFields';
+
+export function decode(keyVal: KeyValue): TapKeySig {
+  if (keyVal.key[0] !== InputTypes.TAP_KEY_SIG) {
```

Comment:
nit pick: check for  `keyVal.key.length ===  1`

Reply:
Fixed.